ui/detailed_results/metrics_helpers.ts

```diff
@@ -60,8 +60,14 @@ export function parseActionMetrics(actionMetricProtos: Array<ActionMetricsProto>
 		.then(name => {
 			if (actionMetric.actionId.tag == 0) {
 				actionMetric.name = name;
-			} else {
```

Comment:
We need to refactor tags, this is really ugly. A different PR though. ;)

Reply:
yes, it should be per-spec instead of global